hack/lib/sync-api.sh

```diff
@@ -26,48 +26,53 @@ echo "git clone"
 cd ..
 git config --global user.email "openyurt-bot@openyurt.io"
 git config --global user.name "openyurt-bot"
-git clone --single-branch --depth 1 git@github.com:openyurtio/yurt-app-manager-api.git yurt-app-manager-api
 
-if [ -d "yurt-app-manager-api/pkg/yurtappmanager/apis" ]
+## sync to code repo
+destRepo=api
+destModule=apps
+
+git clone --single-branch --depth 1 git@github.com:openyurtio/${destRepo}.git ${destRepo}
+
+if [ -d "${destRepo}/pkg/${destModule}/apis" ]
```

Comment:
how about put `api` and `client` under `${destRepo}/${destModule}/apis(client)` folder, remove `pkg` ?